<ui version="4.0" >
 <class>setupDialog</class>
 <widget class="QDialog" name="setupDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Setup</string>
  </property>
  <widget class="QGroupBox" name="headerGroup" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>491</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title" >
    <string>Header</string>
   </property>
   <widget class="QCheckBox" name="checkKeywordsWarn" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Check keywords warn</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="meshDBLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>MeshDB</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="meshDBEdit1" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>50</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="meshDBEdit2" >
    <property name="geometry" >
     <rect>
      <x>300</x>
      <y>50</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>.</string>
    </property>
   </widget>
   <widget class="QLabel" name="includePathLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Include path</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultsDirectoryLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Results directory</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="includePathEdit" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>70</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="resultsDirectoryEdit" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>90</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="simulationGroup" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>140</y>
     <width>491</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title" >
    <string>Simulation</string>
   </property>
   <widget class="QLabel" name="maxOutputLevelLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Max. output level</string>
    </property>
   </widget>
   <widget class="QComboBox" name="maxOutputLevelCombo" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex" >
     <number>3</number>
    </property>
    <item>
     <property name="text" >
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>5</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="coordinateSystemCombo" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>40</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text" >
      <string>Cartesian</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Axi Symmetric</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Cylindric Symmetric</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="coordinateSystemlabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Coordinate system</string>
    </property>
   </widget>
   <widget class="QLabel" name="coordinateMappingLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Coordinate mapping</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="coordinateMappingEdit" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>60</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>1 2 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="simulationTypeLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Simulation type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="simulationTypeCombo" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>80</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text" >
      <string>Steady state</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Transient</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Scanning</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="steadyStateMaxIterLabel" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Steady state max. iter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="steadyStateMaxIterEdit" >
    <property name="geometry" >
     <rect>
      <x>370</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="outputIntervalsLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>100</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Output intervals</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputIntervalsEdit" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>100</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="solverInputFileLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>130</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Solver input file</string>
    </property>
   </widget>
   <widget class="QLabel" name="postFileLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>150</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Post file</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="solverInputFileEdit" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>130</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>skeleton.sif</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="postFileEdit" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>150</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>skeleton.ep</string>
    </property>
   </widget>
   <widget class="QLabel" name="timesteppingMethodLabel" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Timestepping method</string>
    </property>
   </widget>
   <widget class="QLabel" name="bdfOrderLabel" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>BDF order</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeStepIntervalsLabel" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>80</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Timestep intervals</string>
    </property>
   </widget>
   <widget class="QLabel" name="timestepSizesLabel" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>100</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Timestep sizes</string>
    </property>
   </widget>
   <widget class="QComboBox" name="timesteppingMethodCombo" >
    <property name="geometry" >
     <rect>
      <x>370</x>
      <y>40</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text" >
      <string>BDF</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="bdfOrderCombo" >
    <property name="geometry" >
     <rect>
      <x>370</x>
      <y>60</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex" >
     <number>0</number>
    </property>
    <item>
     <property name="text" >
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>5</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="timeStepIntervalsEdit" >
    <property name="geometry" >
     <rect>
      <x>370</x>
      <y>80</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="timestepSizesEdit" >
    <property name="geometry" >
     <rect>
      <x>370</x>
      <y>100</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="constantsGroup" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>330</y>
     <width>491</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title" >
    <string>Constants</string>
   </property>
   <widget class="QLabel" name="gravityLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Gravity</string>
    </property>
   </widget>
   <widget class="QLabel" name="stefanBoltzmannLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Stefan Boltzmann</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gravityEdit" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>0 -1 0 9.82</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="stefanBoltzmannEdit" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>5.67e-08</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="acceptButton" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>410</y>
     <width>491</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>Accept</string>
   </property>
   <property name="icon" >
    <iconset resource="../Mesh3D.qrc" >:/icons/dialog-ok-apply.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Mesh3D.qrc" />
 </resources>
 <connections/>
</ui>
